formsfeeder.client-cli/src/main/java/formsfeeder/client/cli/parameters/CommandLineAppParameters.java

```diff
@@ -163,7 +189,16 @@ private static final AuthParameters asAuthParameters(String authParam) throws Pa
 			return new AbstractMap.SimpleEntry<>(dsParam.substring(0, index).trim(), dsParam.substring(index+1).trim());
 		}
 	}
-	
+
+	private static AbstractMap.SimpleEntry<String, List<String>> asSupplierEntry(String dsParam) {
+		int index = dsParam.indexOf("=");
+		if (index < 0) {
+			return new AbstractMap.SimpleEntry<>(dsParam.trim(), Arrays.asList(""));	// Ideally this would be a strip() instead of trim() under JDK 11
+		} else {
+			return new AbstractMap.SimpleEntry<>(dsParam.substring(0, index).trim(), Arrays.asList(dsParam.substring(index + 1).trim().split(" ")));
+		}
+	}
+

```

Comment:
I get a warning that this routine (`asSupplierEntry()`) is not used.  It's also not covered by the unit tests (not surprisingly, if it's not used).  Can we remove it?